File: C++/moreThanHalfNum.cpp
#include<iostream>
#include<vector>
#include<algorithm>

class Solution{
    public:
    int MoreThanHalfNum_Solution(std::vector<int>& numbers) {
        std::sort(numbers.begin(),numbers.end());
        return numbers[numbers.size()/2];
    }
};

int main(){
    std::vector<int> nums={1,2,3,2,2,2,5,4,2};
    Solution solution;
    std::cout<<solution.MoreThanHalfNum_Solution(nums)<<std::endl;
    return 0;
}

File: python/moreThanHalfNum.py
class Solution:
    def MoreThanHalfNum_Solution(self , numbers: list[int]) -> int:
        numbers.sort()
        return(numbers[len(numbers)//2])
    
nums=[1,2,3,2,2,2,5,4,2]
s=Solution()
print(s.MoreThanHalfNum_Solution(nums))